data/docs_markdown/Monitor_Block_Rewards.md

```
@@ -0,0 +1,108 @@
+The score your worker receives based on its performance as measured by **POW** and **POTL**.
+Rewarded
+The amount of IO Coin the worker earned based on **Total Score**
+#### [](https://docs.io.net/docs/monitor-block-rewards#exceptions)
+**Exempt-Hired**
+If a worker was nominated for a Block Reward, but was hired by a customer during the evaluation hour, io.net exempts the worker from Proof of Work. The exemption occurs so the worker can successfully complete the job for the customer. 
+In the screenshot below, the **IO Worker Block Reward** is marked as **Completed**. The worker was hired by a customer so no POW tasks were assigned. Uptime was tested during the hour and the worker satisfied the requirement. Since it was successful, a reward was granted.
+![](https://files.readme.io/7f01b1e-exempt-hired.png)
+**Exempt-Headnode**
+Headnodes must always be available for jobs. For this reason, POW is not executed against headnodes when nominated for Block Rewards. Uptime (POTL) is tested during the evaluation. In the example below, the headnode satisfied the requirement. Since it was successful, a reward was granted.
+![](https://files.readme.io/b33bbd7-ex_headnode.png)
+9 months ago
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)